app/renderer/components/common/contextMenu/contextMenuItem.js

```diff
@@ -179,7 +179,7 @@ class ContextMenuItem extends ImmutableComponent {
       width: iconSize
     }
 
-    const icon = this.props.contextMenuItem.get('icon')
+    const icon = !this.props.isTor && this.props.contextMenuItem.get('icon')
```

Comment:
small nitpick here that IMO it would be preferrable to have the prop be more semantic, e.g. `shouldDisplayIcon`